lib/parallel_guest_migration_base.pm

```diff
@@ -570,12 +570,21 @@ sub config_host_shared_storage {
             }
             $_temppath = $args{exppath};
             $_temppath =~ s/\//\\\//g;
+            # Create export path directory if it doesn't exist
+            unless (-d $_temppath) {
```

Comment:
I am afraid this
```
unless (-d $_temppath)
```
judges whether ```$_temppath``` exist on worker machine instead of SUT machine. Maybe you can verify your test again with such path on worker machine but not on SUT machine.

Reply:
I am a little confused about it. NFS server just setup on server, not on client. So judges it on server. It is enough.

Reply:
I made test, no such issue.

Reply:
`unless (-d $_temppath)`  is not executed on the test machine, no matter server or client. It runs on where the worker services runs, for your case, void.qam.suse.cz. You should use `script_run("xxx")`. This will be executed on the test machine.

Reply:
if (script_run("test -d $_temppath") != 0) {
}
It also works. will execute on the worker.

Reply:
Explained to Roy offline. Should be clear now.